bundle/edu.gemini.wdba.xmlrpc.server/src/test/scala/edu/gemini/wdba/tcc/TcsConfigTest.scala

```diff
@@ -21,14 +21,6 @@ final class TcsConfigTest extends TestBase {
   }
 
   @Test
-  def testNonSiderealWithHorizonsDesignation(): Unit = {
```

Comment:
I agree that a test case is absolutely needed, but I think we should make a new test file for this with regards to WDBA writing of targets specifically.

I'm thinking something like a `TargetConfigTest`, but I would like to hear back from @swalker2m before I invest any time in this.